tests/test_cli_download_files.py

```diff
@@ -57,27 +63,40 @@ def test_download_files_filter_regexp():
     if os.path.isfile(test_file):
         os.remove(test_file)
     test_download_files_filter = CliRunner()
-    test_result_regexp = test_download_files_filter.invoke(
+    test_result_regexp_one = test_download_files_filter.invoke(
         download_files, ["--recid", 3005, "--filter-regexp", "py$"]
```

Comment:
A good test could be also to get `.cc` files from record 5500, because they are three, and they are not sequential, i.e. this would test whether we can pick up files from random list locations :wink: